<!DOCTYPE html>

<html lang="en">

<head>
    <!-- Google tag (gtag.js) -->
    <script async src="https://www.googletagmanager.com/gtag/js?id=G-CP6MZGJ0DB"></script>
    <script>
      window.dataLayer = window.dataLayer || [];
      function gtag(){dataLayer.push(arguments);}
      gtag('js', new Date());

      gtag('config', 'G-CP6MZGJ0DB');
    </script>

    <meta charset="utf-8" />
    <meta http-equiv="x-ua-compatible" content="ie=edge" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />

    <!-- Define title -->
    <title>Yuhui Hong</title>

    <!-- Include favicon. Use, e.g., https://favicon.io/ -->
    <link rel="icon" type="image/png" href="favicon.png" sizes="32x32" />

    <!-- Include style sheets -->
    <link rel="stylesheet" href="main.css" />

    <!-- Fonts from Google Fonts -->
    <link rel="preconnect" href="https://fonts.googleapis.com" />
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
    <link href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@400;600&display=swap" rel="stylesheet" />

    <!-- Load FontAwesome (for icons) -->
    <link rel="stylesheet"
        href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
        integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
        crossorigin="anonymous" />

    <!-- Load Academicons (for more icons) -->
    <link rel="stylesheet"
        href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.9.2/css/academicons.min.css"
        integrity="sha512-KlJCpRsLf+KKu2VQa5vmRuClRFjxc5lXO03ixZt82HZUk41+1I0bD8KBSA0fY290ayMfWYI9udIqeOWSu1/uZg=="
        crossorigin="anonymous" />
</head>

<body>
    <!-- Navigation Bar -->
    <nav>
      <ul class="navbar">
          <li><a href="index.html">Home</a></li>
          <li><a href="./files/CV_Yuhui.pdf" target="_blank">CV</a></li>
          <li><a href="blogs.html">Blogs</a></li>
      </ul>
    </nav>

    <!-- Main content -->
    <section id="content">
        <!-- Avatar image -->
        <img src="image.jpg" alt="Yuhui Hong" class="avatar" />

        <!-- Text -->
        <h1>Yuhui Hong</h1>
        
        <p>
          Ph.D. candidate in <highlight>Computer Science</highlight>
          <br>
          Indiana University Bloomington
          <br>
          Advised by <a href="https://luddy.indiana.edu/contact/profile/?Haixu_Tang">Prof. Haixu Tang</a>
        </p>
        <p>
          My research explores the intersection of <highlight>deep learning</highlight>, bioinformatics, and cheminformatics, 
          with a focus on advancing the <highlight>identification of small molecules</highlight> in two pathways. 
          The first involves predicting tandem mass spectra and other molecular properties from 3D structures, 
          addressing gaps‚Äîoften referred to as the "dark matter"‚Äîin existing spectral reference libraries. 
          The second approach moves beyond the traditional reliance on database-driven methods by predicting compounds directly from tandem mass spectra. 
          My long-term goal is to use computational and machine learning methods to accelerate <highlight>the discovery of novel drugs and natural products</highlight>.
          Additionally, I am passionate about developing <highlight>reliable and interpretable neural networks</highlight> for real-world applications. 
        </p>
        <p>
            Please feel free to get in touch!
        </p>

        <!-- Icons -->
        <ul class="icons">
          <li>
            <a href="mailto:yuhhong@iu.edu" title="E-Mail"><i class="fas fa-envelope"></i></a>
          </li>
          <li>
            <a href="https://github.com/JosieHong" title="GitHub"><i class="fab fa-github"></i></a>
          </li>
          <li>
            <a href="https://www.linkedin.com/in/yuhui-hong-09871216b/" title="LinkedIn"><i class="fab fa-linkedin"></i></a>
          </li>
          <li>
            <a href="https://scholar.google.com/citations?user=F4wI2p4AAAAJ&hl=en" title="Google Scholar"><i class="ai ai-google-scholar"></i></a>
          </li>
          <li>
            <a href="https://orcid.org/0000-0002-5647-9714"><i class="ai ai-orcid"></i></a>
          </li>
        </ul>
        <hr>

        <!-- Publications Section -->
        <h3>Selected Publications ‚ú®</h3> 
        The complete list of publications can be found on <a href="https://scholar.google.com/citations?user=F4wI2p4AAAAJ&hl=en">[Google Scholar]</a>. 
        
        <h4>Books</h4>
        <div class="paper-card">
          <div class="toc">
            <img src="./imgs/acsinfocus.7e8012.jpg" alt="acsinfocus.7e8012">
          </div>
          <div class="content">
            <div class="title">
              <b>Neural Networks for Chemists</b> </br>
              Qingyang Xiao, Kaiyuan Liu, <u>Yuhui Hong</u> & Haixu Tang (2024). </br>
              <i>American Chemical Society</i>. 
              <a href="https://pubs.acs.org/doi/10.1021/acsinfocus.7e8012">[Primer]</a>
            </div>
            <div class="tldr">
              This primer introduces the basics of neural networks, guiding students, researchers, and professionals to harness their potential. 
              It covers foundational concepts, fully connected networks, advanced architectures, and case studies, illustrating their impact on fields like chemistry, healthcare, and beyond. 
            </div>
          </div>
        </div>

        <h4>Preprint</h4>
        <div class="paper-card">
          <div class="toc">
            <img src="./imgs/fiddle.png" alt="fiddle">
          </div>
          <div class="content">
            <div class="title">
              <b> FIDDLE: a Deep Learning Method for Chemical Formulas Prediction from Tandem Mass Spectra</b> </br>
              <u>Yuhui Hong</u>, Sujun Li, Yuzhen Ye, & Haixu Tang (2024). </br>
              <i>bioRxiv</i> 2024.11.25.625316. 
              <a href="https://www.biorxiv.org/content/10.1101/2024.11.25.625316v1">[Preprint]</a>
              <a href="https://github.com/JosieHong/FIDDLE">[Code]</a>
            </div>
            <div class="tldr">
              FIDDLE (Formula IDentification by Deep LEarning) is introduced as a deep learning-based method for identifying chemical formulas from MS/MS data. 
              It is trained on over 38,000 molecules and 1 million MS/MS spectra collected under various conditions, 
              including collision energy and precursor types, using Quadrupole Time-of-Flight (QTOF) and Orbitrap instruments.
            </div>
          </div>
        </div>

        <h4>Peer-reviewd Articles</h4>
        <div class="paper-card">
          <div class="toc">
            <img src="./imgs/anchem.png" alt="anchem">
          </div>
          <div class="content">
            <div class="title">
              <b>Machine Learning in Small-Molecule Mass Spectrometry</b> </br>
              <u>Yuhui Hong</u>, Yuzhen Ye, & Haixu Tang (2025). </br>
              (Accepted by <i>Annual Review of Analytical Chemistry</i>, to be published on May 2025). 
            </div>
            <div class="tldr">
              This review highlights how machine learning is transforming small molecule mass spectrometry by: 
              (a) predicting MS/MS spectra and properties to expand reference libraries, 
              (b) enhancing spectral matching with automated pattern extraction, and 
              (c) directly predicting molecular structures from MS/MS spectra when reference data is unavailable.
            </div>
          </div>
        </div>
        <div class="paper-card">
          <div class="toc">
            <img src="./imgs/ac.3c04028.jpeg" alt="ac.3c04028">
          </div>
          <div class="content">
            <div class="title">
              <b>Enhanced Structure-Based Prediction of Chiral Stationary Phases for Chromatographic Enantioseparation from 3D Molecular Conformations</b> </br>
              <u>Yuhui Hong</u>, Christopher J Welch, Patrick Piras, & Haixu Tang (2024). </br>
              <i>Analytical Chemistry</i>, 96(6), 2351-2359.
              <a href="https://pubs.acs.org/doi/full/10.1021/acs.analchem.3c04028">[Paper]</a> 
              <a href="https://github.com/JosieHong/3DMolCSP">[Code]</a>
            </div>
            <div class="tldr">
              3DMolCSP leverages a 3D molecular conformation representation algorithm, 
              alongside a dataset of over 300k enantioseparation records. 
              This approach significantly improves enantioselectivity predictions, 
              enabling more efficient and informed decisions in chiral chromatography.
            </div>
          </div>
        </div>
        <div class="paper-card">
          <div class="toc">
            <img src="./imgs/bioinfo.btad354f1.jpeg" alt="bioinfo.btad354f1">
          </div>
          <div class="content">
            <div class="title">
              <b>3DMolMS: Prediction of Tandem Mass Spectra from Three Dimensional Molecular Conformations</b> </br>
              <u>Yuhui Hong</u>, Sujun Li, Christopher J Welch, Shane Tichy, Yuzhen Ye, & Haixu Tang (2023). </br>
              <i>Bioinformatics</i>, btad354. 
              <a href="https://academic.oup.com/bioinformatics/article/39/6/btad354/7186501">[Paper]</a> 
              <a href="https://github.com/JosieHong/3DMolMS">[Code]</a> 
              <a href="https://pypi.org/project/molnetpack/">[PyPI package - molnetpack]</a> 
              <a href="https://spectrumprediction.gnps2.org/">[Web service on GNPS]</a>
              <a href="https://koina.wilhelmlab.org/docs/#post-/3dmolms_qtof/infer">[Inference service on Konia]</a>
            </div>
            <div class="tldr">
              3DMolMS is a deep neural network model that predicts MS/MS spectra from 3D conformations. 
              The learned molecular representation also enhances predictions of chemical properties, 
              such as elution time and collisional cross section, aiding compound identification.
            </div>
          </div>
        </div>
        
        <!-- Talks & Posters Section -->
        <h3>Presentations and Talks üí°</h3>
        <ul class="sublist">
            <li>
              <b>Talk.</b> ‚ÄúEnhanced Structure-Based Prediction of Chiral Stationary Phases for Chromatographic Enantioseparation from 3D Molecular Conformations.‚Äù
              <i>Research Group @ Amgen.</i> Oct. 11, 2024. Virtual talk. 
            </li>
            <li>
              <b>Poster presentation.</b> ‚ÄúPredicting Compositional Fragments of Compounds from Their Tandem Mass Spectra Using
              Deep Neural Networks.‚Äù
              <i>72nd Conference on Mass Spectrometry and Allied Topics.</i> Jun. 2 - 6, 2024. Anaheim, CA. 
              <a href="./files/ASMS24.pdf" target="_blank">[poster]</a>
            </li>
            <li>
              <b>Poster presentation.</b> ‚Äú3DMolMS: Prediction of Tandem Mass Spectra from 3D Molecular Conformations.‚Äù 
              <i>Turkey Run Analytical Chemistry Conference 2023.</i> Sep. 29 - 30, 2023. Marshall, IN.
            </li>
            <li>
              <b>Oral Presentation.</b> ‚ÄúA Machine Learning Model for Chemical Formula Prediction Using Tandem Mass Spectra of Compounds.‚Äù  
              <i>71st Conference on Mass Spectrometry and Allied Topics.</i> Jun. 4 - 8, 2023. Houston, TX.
              <a href="./files/ASMS23.pdf" target="_blank">[slides]</a>
            </li>
            <li>
              <b>Poster presentation.</b> ‚ÄúPrediction of Molecular Tandem Mass Spectra Using 3-Dimensional Conformers.‚Äù 
              <i>70th Conference on Mass Spectrometry and Allied Topics.</i> Jun. 5 - 9, 2022. Minneapolis, MN.
              <a href="./files/ASMS22.pdf" target="_blank">[poster]</a>
            </li>
        </ul>

        <!-- Teaching Section -->
        <h3>Teaching üë©üèΩ‚Äçüè´</h3>
        <ul class="sublist">
          <table border="1">
            <thead>
              <tr>
                  <th>Role</th>
                  <th>Course</th>
                  <th>Name</th>
                  <th>Semester</th>
                  <th>Attachment</th>
              </tr>
            </thead>
            <tbody>
                <tr>
                  <td>Instructor</td>
                  <td>DSCI-D590</td>
                  <td>Topics in Data Science</td>
                  <td>Spring 2025</td>
                  <td></td>
                </tr>
                <tr>
                  <td>Instructor</td>
                  <td>INFO-I529</td>
                  <td>Machine Learning Bioinformatics</td>
                  <td>Fall 2024</td>
                  <td></td>
                </tr>
                <tr>
                  <td>Assistant Instructor</td>
                  <td>DSCI-D351</td>
                  <td>Big Data Analytics</td>
                  <td>Fall 2024 (Aug.-Sep.)</td>
                  <td><a href="./files/D351_Intro_Spark_09032024.pdf" target="_blank">Introduction to Spark</a></td>
                </tr>
                <!-- Add more rows as needed -->
            </tbody>
          </table>
        </ul>
        
        <!-- Professional Services Section -->
        <h3>Professional Services üôå</h3>
        <ul class="sublist">
          <li>Reviewer: 
              <ul class="sublist">
                <li>Journal of Chromatography A</li>
                <li>BMC Genomics</li>
                <li>BMC Bioinformatics</li>
                <li>IEEE Transactions on Computational Biology and Bioinformatics</li>
                <li>Pharmaceutical Research</li>
                <li>Beilstein Journal of Organic Chemistry</li>
                <li>Chemical Physics Letters</li>
              </ul>
          </li>
          <li>Sub-reviewer: 
            <ul class="sublist">
              <li>(Conference) ISMB/ECCB 2025, RECOMB 2025, ACM BCB 2024, ISMB/ECCB 2023, RECOMB 2023, RECOMB 2022; </li>
              <li>(Journal) Analytical Chemistry, International Journal of Mass Spectrometry</li>
              <i>assisted in reviewing papers under the guidance of Prof. Haixu Tang</i>
            </ul>
        </ul>

        <footer>
          <p>&copy; 2024 Yuhui Hong. All rights reserved.</p>
      </footer>
    </section>
</body>

</html>
