<!DOCTYPE html>

<html lang="en">

<head>
    <!-- Google tag (gtag.js) -->
    <script async src="https://www.googletagmanager.com/gtag/js?id=G-CP6MZGJ0DB"></script>
    <script>
      window.dataLayer = window.dataLayer || [];
      function gtag(){dataLayer.push(arguments);}
      gtag('js', new Date());

      gtag('config', 'G-CP6MZGJ0DB');
    </script>

    <meta charset="utf-8" />
    <meta http-equiv="x-ua-compatible" content="ie=edge" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />

    <!-- Define title -->
    <title>Yuhui Hong</title>

    <!-- Include favicon. Use, e.g., https://favicon.io/ -->
    <link rel="icon" type="image/png" href="favicon.png" sizes="32x32" />

    <!-- Include style sheets -->
    <link rel="stylesheet" href="main.css" />

    <!-- Fonts from Google Fonts -->
    <link rel="preconnect" href="https://fonts.googleapis.com" />
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
    <link href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@400;600&display=swap" rel="stylesheet" />

    <!-- Load FontAwesome (for icons) -->
    <link rel="stylesheet"
        href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
        integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
        crossorigin="anonymous" />

    <!-- Load Academicons (for more icons) -->
    <link rel="stylesheet"
        href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.9.2/css/academicons.min.css"
        integrity="sha512-KlJCpRsLf+KKu2VQa5vmRuClRFjxc5lXO03ixZt82HZUk41+1I0bD8KBSA0fY290ayMfWYI9udIqeOWSu1/uZg=="
        crossorigin="anonymous" />
</head>

<body>
    <!-- Navigation Bar -->
    <nav>
      <ul class="navbar">
          <li><a href="index.html">Home</a></li>
          <li><a href="./files/CV_Yuhui.pdf" target="_blank">CV</a></li>
          <li><a href="blogs.html">Blogs</a></li>
      </ul>
    </nav>

    <!-- Main content -->
    <section id="content">
        <!-- Avatar image -->
        <img src="image.jpg" alt="Yuhui Hong" class="avatar" />

        <!-- Text -->
        <h1>Yuhui Hong</h1>
        
        <p>
          Postdoctoral Scholar at <a href="https://noble.gs.washington.edu/">Noble Lab</a> (2025 - Present)
          <br>
          University of Washington
          <br>
          Advised by <a href="https://www.gs.washington.edu/faculty/noble.htm">Professor William Stafford Noble</a>
        </p>
        <p>
          Ph.D. in <highlight>Computer Science</highlight> (2020-2025)
          <br>
          Indiana University Bloomington
          <br>
          Advised by <a href="https://luddy.indiana.edu/contact/profile/?Haixu_Tang">Professor Haixu Tang</a>
        </p>
        <p>
          I am a postdoctoral fellow at the University of Washington, specializing in <highlight>computational mass spectrometry</highlight> 
          and developing methods to analyze biological data from analytical instruments (e.g., LC-MS, GC-MS). 
          Currently working in proteomics mass spectrometry, I focus on advancing peptide <i>de novo</i> 
          sequencing methods to better understand biological systems and enable more comprehensive protein identification.
        </p>
        <p>
          During my PhD studies in Computer Science at Indiana University Bloomington, I developed computational approaches to overcome limitations in small molecule identification. 
          My work focused on moving beyond traditional database-dependent methods by predicting LC-MS/MS spectra and molecular 
          properties from 3D conformations to establish reference libraries for unknown compounds, and by identifying chemical 
          formulas directly from spectra. I also applied deep learning methods to optimize experimental condition settings, 
          improving the efficiency and accuracy of mass spectrometry workflows.
        </p>
        <p>
          I received my B.S. in Computer Science from Xidian University, China, in 2019 
          and worked as a research assistant in computer vision at Xi'an Jiaotong University from 2019 to 2020 
          under the guidance of Professor Yaochen Li. 
          Drawn to the complexity and scientific impact of biological and chemical research, 
          I transitioned to computational biology to apply my technical expertise to problems with direct implications for human health.
        </p>
        <p>
          Please feel free to get in touch!
        </p>

        <!-- Icons -->
        <ul class="icons">
          <li>
            <a href="mailto:yuhhong@uw.edu" title="E-Mail"><i class="fas fa-envelope"></i></a>
          </li>
          <li>
            <a href="https://github.com/JosieHong" title="GitHub"><i class="fab fa-github"></i></a>
          </li>
          <li>
            <a href="https://www.linkedin.com/in/yuhui-hong-09871216b/" title="LinkedIn"><i class="fab fa-linkedin"></i></a>
          </li>
          <li>
            <a href="https://scholar.google.com/citations?user=F4wI2p4AAAAJ&hl=en" title="Google Scholar"><i class="ai ai-google-scholar"></i></a>
          </li>
          <li>
            <a href="https://orcid.org/0000-0002-5647-9714"><i class="ai ai-orcid"></i></a>
          </li>
        </ul>
        <hr>
        
        <!-- News -->
        <h3>News üì∞</h3>
        <p>
          - [09/19/2025] Awarded the <a href="https://escience.washington.edu/people/postdoctoral-fellows/">UW Data Science Fellowship at eScience Institute</a>, University of Washington. 
        </p>
        <p>
          - [07/07/2025] I will join <a href="https://noble.gs.washington.edu/">Noble Lab</a> at University of Washington as a Postdoctoral Scholar in August 2025.
        </p>
        <p>
          - [05/09/2025] Recipient of the the <a href="./files/Luddy_Outstanding_Research_Award_Yuhui.pdf">Luddy Outstanding Research Award</a>. 
        </p>
        <p>
          - [03/04/2025] Our work, <i>A Task-Specific Transfer Learning Approach to Enhancing Small Molecule 
            Retention Time Prediction with Limited Data</i>, has been selected for an 
            <highlight>oral presentation</highlight> at <a href="https://www.asms.org/conferences/annual-conference/annual-conference-homepage">ASMS 2025</a>.
        </p>
        <hr>

        <!-- Publications Section -->
        <h3>Selected Publications ‚ú®</h3> 
        
        <h4>Preprints</h4>
        <div class="paper-card">
          <div class="toc">
            <img src="./imgs/tstl_rt.png" alt="tstl_rt">
          </div>
          <div class="content">
            <div class="title">
              <b> A Task-Specific Transfer Learning Approach to Enhancing Small Molecule Retention Time Prediction with Limited Data</b> </br>
              <u>Yuhui Hong</u>, & Haixu Tang (2025). </br>
              <i>bioRxiv</i> 2025.06.26.661631. 
              <a href="https://www.biorxiv.org/content/10.1101/2025.06.26.661631v1">[Preprint]</a>
              <a href="https://github.com/JosieHong/TSTL_RT">[Code]</a>
            </div>
            <div class="tldr">
              TSTL (Task-Specific Transfer Learning) is introduced as a training strategy for predicting retention times 
              in various LC systems with limited training data. 
              Evaluated across 6 benchmark datasets from different LC systems using 5 deep neural network architectures, 
              TSTL achieved significant improvements in prediction accuracy, 
              increasing average R¬≤ from 0.587 to 0.825 with superior data efficiency.
            </div>
          </div>
        </div>
        
        <h4>Peer-reviewed Articles</h4>
        <div class="paper-card">
          <div class="toc">
            <img src="./imgs/fiddle.png" alt="fiddle">
          </div>
          <div class="content">
            <div class="title">
              <b> FIDDLE: a Deep Learning Method for Chemical Formulas Prediction from Tandem Mass Spectra</b> </br>
              <u>Yuhui Hong</u>, Sujun Li, Yuzhen Ye, & Haixu Tang (2024). </br>
              <i>bioRxiv</i> 2024.11.25.625316. (accepted by Nature Communications). 
              <a href="https://www.biorxiv.org/content/10.1101/2024.11.25.625316v1">[Preprint]</a>
              <a href="https://github.com/JosieHong/FIDDLE">[Code]</a>
              <a href="https://pypi.org/project/msfiddle/">[PyPI package - msfiddle]</a> 
            </div>
            <div class="tldr">
              FIDDLE (Formula IDentification by Deep LEarning) is introduced as a deep learning-based method for identifying chemical formulas from MS/MS data. 
              It is trained on over 38,000 molecules and 1 million MS/MS spectra collected under various conditions, 
              including collision energy and precursor types, using Quadrupole Time-of-Flight (QTOF) and Orbitrap instruments.
            </div>
          </div>
        </div>
        <div class="paper-card">
          <div class="toc">
            <img src="./imgs/ac.3c04028.jpeg" alt="ac.3c04028">
          </div>
          <div class="content">
            <div class="title">
              <b>Enhanced Structure-Based Prediction of Chiral Stationary Phases for Chromatographic Enantioseparation from 3D Molecular Conformations</b> </br>
              <u>Yuhui Hong</u>, Christopher J Welch, Patrick Piras, & Haixu Tang (2024). </br>
              <i>Analytical Chemistry</i>, 96(6), 2351-2359.
              <a href="https://pubs.acs.org/doi/full/10.1021/acs.analchem.3c04028">[Paper]</a> 
              <a href="https://github.com/JosieHong/3DMolCSP">[Code]</a>
            </div>
            <div class="tldr">
              3DMolCSP leverages a 3D molecular conformation representation algorithm, 
              alongside a dataset of over 300k enantioseparation records. 
              This approach significantly improves enantioselectivity predictions, 
              enabling more efficient and informed decisions in chiral chromatography.
            </div>
          </div>
        </div>
        <div class="paper-card">
          <div class="toc">
            <img src="./imgs/bioinfo.btad354f1.jpeg" alt="bioinfo.btad354f1">
          </div>
          <div class="content">
            <div class="title">
              <b>3DMolMS: Prediction of Tandem Mass Spectra from Three Dimensional Molecular Conformations</b> </br>
              <u>Yuhui Hong</u>, Sujun Li, Christopher J Welch, Shane Tichy, Yuzhen Ye, & Haixu Tang (2023). </br>
              <i>Bioinformatics</i>, btad354. 
              <a href="https://academic.oup.com/bioinformatics/article/39/6/btad354/7186501">[Paper]</a> 
              <a href="https://github.com/JosieHong/3DMolMS">[Code]</a> 
              <a href="https://3dmolms.readthedocs.io/en/latest/">[Documentation]</a>
              <a href="https://pypi.org/project/molnetpack/">[PyPI package - molnetpack]</a> 
              <a href="https://koina.wilhelmlab.org/docs/#post-/3dmolms_qtof/infer">[Workflow on Konia]</a>
            </div>
            <div class="tldr">
              3DMolMS is a deep neural network model that predicts MS/MS spectra from 3D conformations. 
              The learned molecular representation also enhances predictions of chemical properties, 
              such as elution time and collisional cross section, aiding compound identification.
            </div>
          </div>
        </div>
        
        <!-- Talks & Posters Section -->
        <h3>Presentations and Talks üí°</h3>
        <ul class="sublist">
            <li>
              <b>Oral Presentation.</b> ‚ÄúA Task-Specific Transfer Learning Approach to Enhancing Small Molecule Retention Time Prediction with Limited Data.‚Äù  
              <i>73rd Conference on Mass Spectrometry and Allied Topics.</i> Jun. 1 - 5, 2025. Baltimore, MD. 
              <a href="./files/ASMS25.pdf" target="_blank">[slides]</a>
            </li>
            <li>
              <b>Talk.</b> ‚ÄúEnhanced Structure-Based Prediction of Chiral Stationary Phases for Chromatographic Enantioseparation from 3D Molecular Conformations.‚Äù
              <i>Research Group @ Amgen.</i> Oct. 11, 2024. Virtual talk. 
            </li>
            <li>
              <b>Poster presentation.</b> ‚ÄúPredicting Compositional Fragments of Compounds from Their Tandem Mass Spectra Using
              Deep Neural Networks.‚Äù
              <i>72nd Conference on Mass Spectrometry and Allied Topics.</i> Jun. 2 - 6, 2024. Anaheim, CA. 
              <a href="./files/ASMS24.pdf" target="_blank">[poster]</a>
            </li>
            <li>
              <b>Poster presentation.</b> ‚Äú3DMolMS: Prediction of Tandem Mass Spectra from 3D Molecular Conformations.‚Äù 
              <i>Turkey Run Analytical Chemistry Conference 2023.</i> Sep. 29 - 30, 2023. Marshall, IN.
            </li>
            <li>
              <b>Oral Presentation.</b> ‚ÄúA Machine Learning Model for Chemical Formula Prediction Using Tandem Mass Spectra of Compounds.‚Äù  
              <i>71st Conference on Mass Spectrometry and Allied Topics.</i> Jun. 4 - 8, 2023. Houston, TX.
              <a href="./files/ASMS23.pdf" target="_blank">[slides]</a>
            </li>
            <li>
              <b>Poster presentation.</b> ‚ÄúPrediction of Molecular Tandem Mass Spectra Using 3-Dimensional Conformers.‚Äù 
              <i>70th Conference on Mass Spectrometry and Allied Topics.</i> Jun. 5 - 9, 2022. Minneapolis, MN.
              <a href="./files/ASMS22.pdf" target="_blank">[poster]</a>
            </li>
        </ul>
        
        <!-- Professional Services Section -->
        <h3>Professional Services üôå</h3>
        <ul class="sublist">
          <li>Reviewer: 
              <ul class="sublist">
                <li>(Conference) ACM BCB 2025</li>
                <li>(Journal) Journal of Chromatography A, 
                  BMC Genomics, 
                  BMC Bioinformatics, 
                  IEEE Transactions on Computational Biology and Bioinformatics, 
                  Pharmaceutical Research, 
                  Beilstein Journal of Organic Chemistry, 
                  Chemical Physics Letters, 
                  PeerJ Computer Science
                </li>
              </ul>
          </li>
          <li>Sub-reviewer: 
            <ul class="sublist">
              <li>(Conference) ISMB/ECCB 2025, RECOMB 2025, ACM BCB 2024, ISMB/ECCB 2023, RECOMB 2023, RECOMB 2022; </li>
              <li>(Journal) Analytical Chemistry, International Journal of Mass Spectrometry</li>
              <i>assisted in reviewing papers under the guidance of Prof. Haixu Tang</i>
            </ul>
          </li>
        </ul>

        <!-- Teaching Section -->
        <h3>Teaching üë©üèΩ‚Äçüè´</h3>
        <ul class="sublist">
          <table border="1">
            <thead>
              <tr>
                  <th>Role</th>
                  <th>Course</th>
                  <th>Name</th>
                  <th>Semester</th>
                  <th>Attachment</th>
              </tr>
            </thead>
            <tbody>
                <tr>
                  <td>Course Designer</td>
                  <td>DSCI-D590</td>
                  <td>AI on Ramp</td>
                  <td>From Fall 2025 onward</td>
                  <td></td>
                </tr>
                <tr>
                  <td>Instructor</td>
                  <td>DSCI-D590</td>
                  <td>Topics in Data Science</td>
                  <td>Spring 2025</td>
                  <td></td>
                </tr>
                <tr>
                  <td>Instructor</td>
                  <td>INFO-I529</td>
                  <td>Machine Learning Bioinformatics</td>
                  <td>Fall 2024</td>
                  <td></td>
                </tr>
                <tr>
                  <td>Assistant Instructor</td>
                  <td>DSCI-D351</td>
                  <td>Big Data Analytics</td>
                  <td>Fall 2024 (Aug.-Sep.)</td>
                  <td><a href="./files/D351_Intro_Spark_09032024.pdf" target="_blank">Introduction to Spark</a></td>
                </tr>
                <!-- Add more rows as needed -->
            </tbody>
          </table>
        </ul>

        <footer>
          <p>&copy; 2024 Yuhui Hong. All rights reserved.</p>
      </footer>
    </section>
</body>

</html>
