<!DOCTYPE html>

<html lang="en">

<head>
    <!-- Google tag (gtag.js) -->
    <script async src="https://www.googletagmanager.com/gtag/js?id=G-CP6MZGJ0DB"></script>
    <script>
      window.dataLayer = window.dataLayer || [];
      function gtag(){dataLayer.push(arguments);}    
      gtag('js', new Date());

      gtag('config', 'G-CP6MZGJ0DB');
    </script>

    <meta charset="utf-8" />
    <meta http-equiv="x-ua-compatible" content="ie=edge" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />

    <!-- Define title -->
    <title>Research — Yuhui Hong</title>

    <!-- Include favicon. -->
    <link rel="icon" type="image/png" href="favicon.png" sizes="32x32" />

    <!-- Include style sheets -->
    <link rel="stylesheet" href="main.css" />

    <!-- Fonts from Google Fonts -->
    <link rel="preconnect" href="https://fonts.googleapis.com" />
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
    <link href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@400;600&display=swap" rel="stylesheet" />

    <!-- Load FontAwesome (for icons) -->
    <link rel="stylesheet"
        href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
        crossorigin="anonymous" />

    <!-- Load Academicons -->
    <link rel="stylesheet"
        href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.9.2/css/academicons.min.css"
        crossorigin="anonymous" />
</head>

<body>
    <!-- Navigation Bar -->
    <nav>
        <ul class="navbar">
                <li><a href="index.html">Home</a></li>
                <li><a href="research.html">Research</a></li>
                <li><a href="talks.html">Talks</a></li>
                <li><a href="professional_service.html">Professional Service</a></li>
                <li><a href="teaching.html">Teaching</a></li>
                <li><a href="blogs.html">Blogs</a></li>
                <li><a href="./files/CV_Yuhui.pdf" target="_blank">CV</a></li>
        </ul>
    </nav>

    <section id="content">
        <h2>Research</h2>

        <h3>Machine Learning for Small Molecule Mass Spectrometry and Chromatography</h3>
        
        <div class="paper-card">
          <div class="toc">
            <img src="./imgs/tstl_rt.png" alt="tstl_rt">
          </div>
          <div class="content">
            <div class="title">
              <b> A Task-Specific Transfer Learning Approach to Enhancing Small Molecule Retention Time Prediction with Limited Data</b> </br>
              <u>Yuhui Hong</u>, & Haixu Tang (2025). </br>
              <i>bioRxiv</i> 2025.06.26.661631. 
              <a href="https://www.biorxiv.org/content/10.1101/2025.06.26.661631v1">[Preprint]</a>
              <a href="https://github.com/JosieHong/TSTL_RT">[Code]</a>
            </div>
            <div class="tldr">
              TSTL (Task-Specific Transfer Learning) is introduced as a training strategy for predicting retention times 
              in various LC systems with limited training data. 
              Evaluated across 6 benchmark datasets from different LC systems using 5 deep neural network architectures, 
              TSTL achieved significant improvements in prediction accuracy, 
              increasing average R² from 0.587 to 0.825 with superior data efficiency.
            </div>
          </div>
        </div>
        
        <div class="paper-card">
          <div class="toc">
            <img src="./imgs/fiddle.png" alt="fiddle">
          </div>
          <div class="content">
            <div class="title">
              <b> FIDDLE: a Deep Learning Method for Chemical Formulas Prediction from Tandem Mass Spectra</b> </br>
              <u>Yuhui Hong</u>, Sujun Li, Yuzhen Ye, & Haixu Tang (2024). </br>
              <i>bioRxiv</i> 2024.11.25.625316. (accepted by Nature Communications). 
              <a href="https://www.biorxiv.org/content/10.1101/2024.11.25.625316v1">[Preprint]</a>
              <a href="https://github.com/JosieHong/FIDDLE">[Code]</a>
              <a href="https://pypi.org/project/msfiddle/">[PyPI package - msfiddle]</a> 
            </div>
            <div class="tldr">
              FIDDLE (Formula IDentification by Deep LEarning) is introduced as a deep learning-based method for identifying chemical formulas from MS/MS data. 
              It is trained on over 38,000 molecules and 1 million MS/MS spectra collected under various conditions, 
              including collision energy and precursor types, using Quadrupole Time-of-Flight (QTOF) and Orbitrap instruments.
            </div>
          </div>
        </div>

        <div class="paper-card">
            <div class="toc">
                <img src="./imgs/ac.3c04028.jpeg" alt="ac.3c04028">
            </div>
            <div class="content">
                <div class="title">
                <b>Enhanced Structure-Based Prediction of Chiral Stationary Phases for Chromatographic Enantioseparation from 3D Molecular Conformations</b> </br>
                <u>Yuhui Hong</u>, Christopher J Welch, Patrick Piras, & Haixu Tang (2024). </br>
                <i>Analytical Chemistry</i>, 96(6), 2351-2359.
                <a href="https://pubs.acs.org/doi/full/10.1021/acs.analchem.3c04028">[Paper]</a> 
                <a href="https://github.com/JosieHong/3DMolCSP">[Code]</a>
                </div>
                <div class="tldr">
                3DMolCSP leverages a 3D molecular conformation representation algorithm, 
                alongside a dataset of over 300k enantioseparation records. 
                This approach significantly improves enantioselectivity predictions, 
                enabling more efficient and informed decisions in chiral chromatography.
                </div>
            </div>
        </div>

        <div class="paper-card">
            <div class="toc">
                <img src="./imgs/bioinfo.btad354f1.jpeg" alt="bioinfo.btad354f1">
            </div>
            <div class="content">
                <div class="title">
                <b>3DMolMS: Prediction of Tandem Mass Spectra from Three Dimensional Molecular Conformations</b> </br>
                <u>Yuhui Hong</u>, Sujun Li, Christopher J Welch, Shane Tichy, Yuzhen Ye, & Haixu Tang (2023). </br>
                <i>Bioinformatics</i>, btad354. 
                <a href="https://academic.oup.com/bioinformatics/article/39/6/btad354/7186501">[Paper]</a> 
                <a href="https://github.com/JosieHong/3DMolMS">[Code]</a> 
                <a href="https://3dmolms.readthedocs.io/en/latest/">[Documentation]</a>
                <a href="https://pypi.org/project/molnetpack/">[PyPI package - molnetpack]</a> 
                <a href="https://koina.wilhelmlab.org/docs/#post-/3dmolms_qtof/infer">[Workflow on Konia]</a>
                </div>
                <div class="tldr">
                3DMolMS is a deep neural network model that predicts MS/MS spectra from 3D conformations. 
                The learned molecular representation also enhances predictions of chemical properties, 
                such as elution time and collisional cross section, aiding compound identification.
                </div>
            </div>
        </div>

        <h3>Computational Methods for Proteomics Mass Spectrometry</h3>
        <div class="paper-card">
            <div class="toc">
                <img src="./imgs/koina.png" alt="koina">
            </div>
            <div class="content">
                <div class="title">
                <b>Koina: Democratizing machine learning for proteomics research</b> </br>
                Ludwig Lautenbacher, Kevin L Yang, Tobias Kockmann, Christian Panse, 
                Wassim Gabriel, Dulguun Bold, Elias Kahl, Matthew Chambers, 
                Brendan X MacLean, Kai Li, Fengchao Yu, Brian C Searle, 
                Damien Beau Wilburn, Mohammad Reza Zare Shahneh, <u>Yuhui Hong</u>, Haixu Tang, 
                Mingxun Wang, Ralf Gabriels, Robbin Bouwmeester, Robbe Devreese, Jesse Angelis, 
                Eduard Sabidó, Tobias K Schmidt, Alexey I Nesvizhskii, Mathias Wilhelm (2025).</br>
                <i>Nature Communications</i>, 16(1), 9933.
                <a href="https://www.nature.com/articles/s41467-025-64870-5">[Paper]</a> 
                <a href="https://zenodo.org/records/16800964">[Code]</a> 
                </div>
                <div class="tldr">
                Koina is a user-friendly platform that enables proteomics researchers to apply machine learning without coding expertise. 
                It offers pre-configured workflows for common tasks like, tandem mass spectra, retention time and collisional cross section prediction, 
                along with customizable options for advanced users.
                </div>
            </div>
        </div>
        

        <h3>Reliable and Transparent Microbiome-Based Host Phenotype Prediction **</h3>

        * <i>equal contribution as first author</i><br>
        ** <i>this direction of work was conducted with colleraboration from Yuzhen Ye's lab</i><br>

        <div class="paper-card">
            <div class="toc">
                <img src="./imgs/microkpnn_cf.png" alt="microkpnn_cf">
            </div>
            <div class="content">
                <div class="title">
                <b>Confounder-free predictive models for microbiome-based host phenotype prediction</b> </br>
                Mahsa Monshizadeh*, <u>Yuhui Hong*</u>, Yuzhen Ye (2024). </br>
                <i>bioRxiv</i>, 2025.01.29.635502.
                <a href="https://www.biorxiv.org/content/10.1101/2025.01.29.635502v1.abstract">[Paper]</a> 
                <a href="https://github.com/mgtools/MicrobiomeCF">[Code]</a> 
                </div>
                <div class="tldr">
                Confounding factors like medications can severely bias microbiome-based disease predictions, 
                leading to spurious associations. This study developed confounder-free models using adversarial optimization 
                to remove biases while preserving true phenotype-microbiome associations. 
                Tested on type 2 diabetes data with metformin as confounder, 
                both FNN_CF and MicroKPNN_CF outperformed conventional approaches by identifying genuine disease markers. 
                MicroKPNN_CF offered superior interpretability despite slightly lower accuracy than FNN_CF.
                </div>
            </div>
        </div>

        <div class="paper-card">
            <div class="toc">
                <img src="./imgs/microkpnn_mt.jpeg" alt="microkpnn_mt">
            </div>
            <div class="content">
                <div class="title">
                <b>Multitask knowledge-primed neural network for predicting missing metadata and host phenotype based on human microbiome</b> </br>
                Mahsa Monshizadeh*, <u>Yuhui Hong*</u>, Yuzhen Ye (2024). </br>
                <i>Bioinformatics Advances</i>, vbae203. 
                <a href="https://academic.oup.com/bioinformaticsadvances/article/5/1/vbae203/7923434">[Paper]</a> 
                <a href="https://github.com/mgtools/MicroKPNN-MT">[Code]</a> 
                </div>
                <div class="tldr">
                Metadata like age and gender are often missing in microbiome studies but crucial for accurate disease prediction. 
                MicroKPNN-MT addresses this by either using available metadata as input or predicting it from microbiome profiles. 
                Tested across 25 diseases, 
                the model showed that incorporating real or predicted metadata improves both prediction accuracy and generalizability, 
                making it a practical tool for microbiome-based disease prediction with incomplete data.
                </div>
            </div>
        </div>

        <footer>
          <p>&copy; 2025 Yuhui Hong. All rights reserved.</p>
        </footer>
    </section>
</body>

</html>
